### PR TITLE
TRON-1859: Updates date arithmetic to handle whitespace

### PR DESCRIPTION
**Pull request by bobokun**
```
>>> from tron.utils.timeutils import DateArithmetic
>>> da = DateArithmetic()
>>> da.parse('shortdate - 1')
'2023-03-08'
>>> da.parse('shortdate-1')
'2023-03-08'
>>> da.parse('shortdate - 2')
'2023-03-07'
```

### CODE REVIEW

File: tron/utils/timeutils.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def parse(cls, date_str, dt=None):
         based on date format (Ex: seconds for unixtime, days for day).
         """
         dt = dt or current_time()
-
+        date_str = date_str.replace(" ", "")
```
Comment (nemacysts) [inline tron/utils/timeutils.py:96]: this should probably be a regex replace for whitespace - otherwise, `'shortdate -  1' would still fail to parse

Comment (bobokun) [reply tron/utils/timeutils.py:96]: hm, I'm not sure how 'shortdate - 1' would fail to parse with my logic. I try not to use regex since it's slower 😅

Comment (nemacysts) [reply tron/utils/timeutils.py:96]: ah - I was just mis-remembering how `replace()` worked :p (for some reason I had it in my head that it wouldn't replace multiple consecutive spaces)

**Review by nemacysts — APPROVED** [review]

File: tron/utils/timeutils.py (hunk 1 above)
Comment (nemacysts) [reply tron/utils/timeutils.py:96]: (in other words, my brain stops working after 5pm 🤣 )

**Pull request merged**